lib/elix/brain/store.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Elix.Brain.Store do
+  @callback start_link :: any
```

Comment:
Haven't seen `@callback` before AFAIK. What does that do? This looks dangerously like a header file 😁